spec/controllers/verify/confirmations_controller_spec.rb

```diff
@@ -66,6 +66,7 @@ def stub_idv_session
 
     context 'user used 2FA phone as phone of record' do
       before do
+        subject.idv_session.params['phone_confirmed_at'] = Time.now
```

Comment:
When the 2FA phone and the phone of record match, `phone_confirmed_at` is set [here](https://github.com/18F/identity-idp/blob/4f94d0ef30218f219f14f4a1a81e90e2270cc10e/app/forms/idv/phone_form.rb#L39)